src/components/CookieWarning.js

```diff
@@ -0,0 +1,56 @@
+import _ from 'lodash/fp'
+import { useState } from 'react'
+import { div, h } from 'react-hyperscript-helpers'
+import { ButtonPrimary, ButtonSecondary, Link } from 'src/components/common'
+import { Ajax } from 'src/libs/ajax'
+import { signOut } from 'src/libs/auth'
+import colors from 'src/libs/colors'
+import * as Nav from 'src/libs/nav'
+import { getLocalPref, setLocalPref } from 'src/libs/prefs'
+import { authStore } from 'src/libs/state'
+import * as Utils from 'src/libs/utils'
+
+
+export const cookiesAcceptedKey = 'cookiesAccepted'
+
+export const CookieWarning = () => {
```

Comment:
how can I test this? what happens if I never hit accept or reject and try to use the site?

Reply:
It works as planned. To test this, spin up a local UI server on this branch (`yarn install`, and then `yarn start` in the terra-ui directory)

If they never hit accept or reject, it leaves the popup open, but we consider that tacit acceptance of cookies.